src/Wallabag/CoreBundle/Event/Subscriber/NewArticleHookSubscriber.php

```diff
@@ -13,10 +13,12 @@
 
 class NewArticleHookSubscriber implements EventSubscriberInterface
 {
+    private $client;
     private $logger;
 
     public function __construct(LoggerInterface $logger)
     {
+        $this->client = new \GuzzleHttp\Client;
```

Comment:
Please no, inject it in your services.yml definition. So we can easily inject a mock in test.